packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -183,34 +183,40 @@ const generateLiveMessage = (props: ChatMessageContentProps): string => {
 };
 
 const messageContentAriaText = (props: ChatMessageContentProps): string | undefined => {
-  // Strip all html tags from the content for aria.
+  if (props.message.content) {
+    // Replace all <img> tags with 'image' for aria.
+    const parsedContent = DOMPurify.sanitize(props.message.content, {
+      ALLOWED_TAGS: ['img'],
+      RETURN_DOM_FRAGMENT: true
+    });
 
-  return props.message.content
-    ? props.message.mine
+    parsedContent.childNodes.forEach((child) => {
+      if (child.nodeName !== 'IMG') {
```

Comment:
should this be case-insensitive?

Reply:
> should this be case-insensitive?

I've tried 'img' and it actually doesn't work, but I guess it doesn't hurt to make it case-insensitive.